.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ workflows:
           matrix:
             parameters:
               python-version: &python-versions [3.6.8, 3.7.9, 3.8.9, 3.9.4]
-              dimod-version: [==0.8.0, <0.10, '[preprocessing]<0.11']
+              dimod-version: [==0.8.0, <0.9, <0.10, <0.11]
```

Comment:
Do we need 0.8.0?

Reply:
Eh, why not? It would be proactive work to remove it and that's currently the bottom of the range.